packages/teams-js/src/public/tasks.ts

```diff
@@ -126,12 +126,18 @@ export namespace tasks {
   }
 
   /**
+   * @hidden
```

Comment:
I am not sure why functions like this and `getBotUrlDialogInfoFromTaskInfo` and `getUrlDialogInfoFromTaskInfo` are not marked internal and hidden. We do not expect users to call them directly. These are just the helper functions. 
Is it a major change to mark them hidden and internal now?

Reply:
sounds good! Another point is it is already deprecated. So, there is no need to release a major change for this.